docs/scos/dev/front-end-development/202212.0/oryx/styling/colors.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Oryx: Color system"
+description: Color design tokens are used for a consistent design system through the components in Oryx applications
+last_updated: July 9, 2023
+template: concept-topic-template
+---
+
+An important part of the application user interface are the colors. Colors are used everywhere throughout components and are important for a good user experience. To ensure that you can adjust the colors to your needs throughout the application, a configurable color system is provided.
+
+The color system allows to setup the color values globally as well as override them per component. Components do not define _values_ for colors directly in their CSS, but use _design tokens_. Design tokens are (CSS) variables that are provided by themes, which you can customize in your project.
+
+The color system comes with semantic color types and a consistent number of color steps. This results in a consistent naming system throughout all the components. The actual colors for those groups are not relevant for the color system and are part of the theme configuration. This allows you to configure the colors in a global theme.
+
+## The color palette
+
+The color palette provides a number of colors that you can use to quickly setup an Oryx application with the color of your choice. This is particular useful to demonstrate and experience how consistent the color system is applied cross all components. All colors in the color palette are fully accessible by default, both in light and dark mode.
+
+The color palette contains all the colors provided by the [Radix color system](https://www.radix-ui.com/docs/colors/overview/installation), an open source colors system. Additionally, a "Spryker green" and "Spryker gray" is added.
```

Comment:
Not sure installation is the best link to put here, maybe it's better to change this link to usage - https://www.radix-ui.com/docs/colors/overview/usage?